### PR TITLE
Fix "<<" datastore modification

### PR DESCRIPTION
**Pull request by wvu**
First batch, second try.

I can use `+` instead of `#{}` if preferred. A bit easier to read. :)
#4296

### CODE REVIEW

**Comment by wvu-r7** [comment]
I can use ```+``` instead of ```#{}``` and ```%Q``` if preferred. A bit easier to read. I <3 my Perlisms, though. :)

File: modules/auxiliary/fuzzers/dns/dns_fuzzer.rb [hunk 1]
```diff
@@ -329,25 +329,6 @@ def dns_send(data,method)
     end
   end
 
-  def fix_variables
```
Comment (todb-r7) [inline modules/auxiliary/fuzzers/dns/dns_fuzzer.rb]: So, removing this pretty much just changes the behavior completely, no?

**Comment by todb-r7** [comment]
@wvu-r7 take a look at wvu-r7#10, other than that, this looks good to go.

**Comment by wvu-r7** [comment]
Thanks, @todb-r7! Class variables. ;)

**Comment by todb-r7** [comment]
Well this makes me happy:

````
Merge made by the 'recursive' strategy.
 modules/auxiliary/admin/http/nexpose_xxe_file_read.rb          |  2 +-
 modules/auxiliary/fuzzers/dns/dns_fuzzer.rb                    | 37 ++++++++++++++++---------------------
 modules/exploits/multi/http/zabbix_script_exec.rb              |  2 +-
 modules/exploits/windows/browser/real_arcade_installerdlg.rb   |  2 +-
 modules/exploits/windows/browser/zenworks_helplauncher_exec.rb |  2 +-
 modules/exploits/windows/scada/scadapro_cmdexe.rb              |  2 +-
 6 files changed, 21 insertions(+), 26 deletions(-)
[*] Running msftidy.rb in .git/hooks/post-merge mode
--- Checking new and changed module syntax with tools/msftidy.rb ---
modules/auxiliary/admin/http/nexpose_xxe_file_read.rb - msftidy check passed
modules/auxiliary/fuzzers/dns/dns_fuzzer.rb - msftidy check passed
modules/exploits/multi/http/zabbix_script_exec.rb - msftidy check passed
modules/exploits/windows/browser/real_arcade_installerdlg.rb - msftidy check passed
modules/exploits/windows/browser/zenworks_helplauncher_exec.rb - msftidy check passed
modules/exploits/windows/scada/scadapro_cmdexe.rb - msftidy check passed

````

**Pull request merged**